### DISCUSSION TITLE
Highlighting --help

### DISCUSSION THREAD

**Discussion by aaaaaadrian** [sharkdp/bat#2672] [Q&A]
Hey pals!

I just found the `Highlighting --help messages` and I love the way it styles everything.

I added the following to my .zshrc

```
alias -g -- -h='-h 2>&1 | bat --language=help --style=plain'
alias -g -- --help='--help 2>&1 | bat --language=help --style=plain'
```

When I reload the config, I get an error with an antidote ohmyzsh package.

The error is ~/Library/Caches/antidote/https-COLON--SLASH--SLASH-github.com-SLASH-ohmyzsh-SLASH-ohmyzsh/lib/diagnostics.zsh:134: parse error near `>&`

The section it is referring to is here on line 134:

```
 121   │   # Installed programs
 122   │   programs=(sh zsh ksh bash sed cat grep ls find git posh)
 123   │   local progfile="" extra_str="" sha_str=""
 124   │   for program in $programs; do
 125   │     extra_str="" sha_str=""
 126   │     progfile=$(builtin which $program)
 127   │     if [[ $? == 0 ]]; then
 128   │       if [[ -e $progfile ]]; then
 129   │         if builtin whence shasum &>/dev/null; then
 130   │           sha_str=($(command shasum $progfile))
 131   │           sha_str=$sha_str[1]
 132   │           extra_str+=" SHA $sha_str"
 133   │         fi
 134   │         if [[ -h "$progfile" ]]; then
 135   │           extra_str+=" ( -> ${progfile:A} )"
 136   │         fi
 137   │       fi
 138   │       builtin printf '%-9s %-20s %s\n' "$program is" "$progfile" "$extra_str"
 139   │     else
 140   │       builtin echo "$program: not found"
 141   │     fi
 142   │   done
```

Is there something I can do to modify either the antidote diagnostic file? Or is this just as innocuous as it gets?

**Comment by keith-hall** [comment]
Hi, I don't know the answer (I'm not a `zsh` user), but it seems some people have successfully used a similar alias successfully: https://github.com/sharkdp/bat/pull/2619/files
